packages/bap-ida/bap-ida.1.0.0~alpha/opam

```diff
@@ -24,6 +24,7 @@ remove: [
 ]
 depends: [
          "conf-ida"
+         "bap-ida-python"
```

Comment:
Either I'm missing something, but why does it depend on `core_kernel`, `oasis` and `ppx_jane`?